src/v/config/configuration.cc

```diff
@@ -3526,6 +3526,12 @@ configuration::configuration()
       "Normalize schemas as they are read from the topic on startup.",
       {.needs_restart = needs_restart::yes, .visibility = visibility::user},
       false)
+  , schema_registry_protobuf_renderer_v2(
+      *this,
+      "schema_registry_protobuf_renderer_v2",
+      "Enables experimental protobuf renderer to support normalize=true.",
```

Comment:
should this be a `experimental property`?

Reply:
> should this be a `experimental property`?

I decided against it because your cover note suggested this wasn't the use case.

I want field testing before enabling it.

Reply:
i guess i'm confused about "experimental" in the description. this suggests that it's something that will be shipped and people should be careful about turning on. i think that would be a use case for experimental property (field would have to enable it before testing). but yeh not exactly sure what you read in the cover letter that indicated it wasn't the proper use case. maybe it isn't

Reply:
> i guess i'm confused about "experimental" in the description. this suggests that it's something that will be shipped and people should be careful about turning on. i think that would be a use case for experimental property (field would have to enable it before testing). but yeh not exactly sure what you read in the cover letter that indicated it wasn't the proper use case. maybe it isn't

["Note that this work is intended for development only. It is not intended for managing access to beta features, or any feature that is visible in production."](https://github.com/redpanda-data/redpanda/pull/23392)

Reply:
thanks. i mean, mechanically i think it might handle the use case, so it could make sense to consider expanding the scope of what it was originally intended for.